src/containers/Reports/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Sidebar from 'Components/Sidebar';
+import privateRoute from 'Hocs/privateRoute';
+
+import './index.scss';
+
+function reports() {
+  return (
+    <div className="reports">
+      <Sidebar />
+      <div className="reports__content">
+        soy los reports <span className="reports__content--wi">WIIIIIIIIIIIIIIIIIII</span>!!!!!!!!
+      </div>
+    </div>
+  );
+}
+
+export default privateRoute(reports);
```

Comment:
Acordate de usar `arrow function`, y de nombrar los componentes con la letra inicial en mayuscula (`Reports`).
```suggestion
import React from 'react';

import Sidebar from 'Components/Sidebar';
import privateRoute from 'Hocs/privateRoute';

import './index.scss';

const Reports = () => (
  <div className="reports">
    <Sidebar />
    <div className="reports__content">
      soy los reports <span className="reports__content--wi">WIIIIIIIIIIIIIIIIIII</span>!!!!!!!!
    </div>
  </div>
);

export default privateRoute(Reports);
```